ament_flake8/ament_flake8/main.py

```diff
@@ -70,6 +70,14 @@ def main(argv=sys.argv[1:]):
         print("Could not find config file '%s'" % args.config_file, file=sys.stderr)
         return 1
 
+    # Filter out folders having AMENT_IGNORE or COLCON_IGNORE
+    if args.excludes is None:
+        args.excludes = []
+    for dirpath, dirnames, filenames in os.walk(os.getcwd()):
+        if 'AMENT_IGNORE' in filenames or 'COLCON_IGNORE' in filenames:
```

Comment:
I don't think it makes sense to ignore directories with `COLCON_IGNORE` since ament is colcon agnostic.

The other linters similarly only check for `AMENT_IGNORE`.

Reply:
I agree, you should just use `AMENT_IGNORE` instead of `COLCON_IGNORE`.

Reply:
@jacobperron @wjwwood Thanks for pointing it out. I've removed the `COLCON_IGNORE`. Please review again.